lib/models/emailAction.js

```diff
@@ -52,8 +52,13 @@ function SendMail(action, event, callback) {
 
         transporter.sendMail(mailOptions, function(err, info) {
             logger.debug('emailAction.SendMail %j %j %j', mailOptions, err, info);
+            // Not a HTTP request, must count outgoingTransaction
```

Comment:
As discussed by skype, this (and the other two similar comments) could use:

// Not a HTTP request, so outgoingTransacion hasn't already counted and must be counted now